robottelo/constants.py

```diff
@@ -278,6 +278,7 @@
 FAKE_2_YUM_REPO = "http://inecas.fedorapeople.org/fakerepos/zoo2/"
 FAKE_3_YUM_REPO = "http://omaciel.fedorapeople.org/fakerepo01"
 FAKE_4_YUM_REPO = "http://omaciel.fedorapeople.org/fakerepo02"
+FAKE_5_YUM_REPO = "http://{0}:{1}@rplevka.fedorapeople.org/fakerepo01/"
```

Comment:
We definitely need a shared Fedora People account so we can centralize every data we need. Off course this is not for this PR.